src/runner.ts

```diff
@@ -87,16 +87,37 @@ export class ActionRunner {
             }
           }
           if (reports.length > 0) {
-            const finalReport: RuleReport = {
-              missingReviews: reports.reduce((a, b) => a + b.missingReviews, 0),
-              missingUsers: [...new Set(reports.flatMap((r) => r.missingUsers))],
-              teamsToRequest: [...new Set(reports.flatMap((r) => r.teamsToRequest ?? []))],
-              usersToRequest: [...new Set(reports.flatMap((r) => r.usersToRequest ?? []))],
-              name: rule.name,
-            };
+            const finalReport = unifyReport(reports, rule.name);
             this.logger.error(`Missing the reviews from ${JSON.stringify(finalReport.missingUsers)}`);
             errorReports.push(finalReport);
           }
+        } else if (rule.type === "or") {
+          const reports: ReviewReport[] = [];
+          for (const reviewer of rule.reviewers) {
+            const [result, missingData] = await this.evaluateCondition(reviewer);
+            if (result) {
+              // This is a OR condition, so with ONE positive result
+              // we can continue the loop to check the following rule
+              continue ruleCheck;
```

Comment:
If we're satisfied with one passed check, why do we need to continue evaluating the rules?